p2p/discover/v4_udp.go

```diff
@@ -126,14 +144,15 @@ type (
 	}
 )
 
-// packet is implemented by all v4 protocol messages.
+// packetV4 is implemented by all v4 protocol messages.
 type packetV4 interface {
 	// preverify checks whether the packet is valid and should be handled at all.
 	preverify(t *UDPv4, from *net.UDPAddr, fromID enode.ID, fromKey encPubkey) error
 	// handle handles the packet.
 	handle(t *UDPv4, from *net.UDPAddr, fromID enode.ID, mac []byte)
-	// name returns the name of the packet for logging purposes.
-	name() string
+	// packet name and type for logging purposes.
+	packetName() string
+	packetType() byte
```

Comment:
Nit, usually the Go stdlib uses `kind` in these scenarios, so you can have `packetV4.name` and `packetV4.kind` instead of `packetV4.packetName` and `packetV4.packetType`.